docs/how-to/peering-with-content-providers.md

```diff
@@ -95,43 +82,31 @@ This list is provided for informational purposes only. The IPFS Project does not
 |`12D3KooWSW4hoHmDXmY5rW7nCi9XmGTy3foFt72u86jNP53LTNBJ`|`/ip4/5.161.55.227/tcp/4001`<br/>`/ip4/5.161.55.227/udp/4001/quic`<br/>`/ip6/2a01:4ff:f0:1e5a::1/tcp/4001`<br/>`/ip6/2a01:4ff:f0:1e5a::1/udp/4001/quic`|
 |`12D3KooWSDj6JM2JmoHwE9AUUwqAFUEg9ndd3pMA8aF2bkYckZfo`|`/ip4/5.161.92.36/tcp/4001`<br/>`/ip4/5.161.92.36/udp/4001/quic`<br/>`/ip6/2a01:4ff:f0:3764::1/tcp/4001`<br/>`/ip6/2a01:4ff:f0:3764::1/udp/4001/quic`|
 
-### Textile
+### StorryTV
 
 | Peer ID | Addresses |
 | ------- | --------- |
-|`QmR69wtWUMm1TWnmuD4JqC1TWLZcc8iR2KrTenfZZbiztd`|`/ip4/104.210.43.77`|
+|`bafzaajaiaejca3eporu7skzkukuie5nopngnubidqmnf57fl3qet7angukq234qj`|`/dnsaddr/peer.ipfs.storry.tv`|
+|`bafzaajaiaejcajedrki5t7qmgxzhwr24pfxb5cwrdf5d7rktguqurqztxsbffbvh`|`/dnsaddr/peer2.ipfs.storry.tv`|
 
 ### Web3.Storage
 
-See [web3.storage/PEERS](https://github.com/web3-storage/web3.storage/blob/main/PEERS).
-
-### 8api.sh
-
-| Peer ID | Addresses |
-| ------- | --------- |
-|`12D3KooWGASC2jm3pmohEJXUhuStkxDitPgzvs4qMuFPaiD9x1BA`|`/ip4/78.46.108.24`|
-|`12D3KooWRbWZN3GvLf9CHmozq4vnTzDD4EEoiqtRJxg5FV6Gfjmm`|`/ip4/65.109.19.136`|
-
-### Spheron Network
-
 | Peer ID | Addresses |
 | ------- | --------- |
-|`12D3KooWFdciAga3zq3TZdrmDospFNrWRALe6VM6JpoKGEfVrqEi`|`/ip4/5.199.162.230`|
-|`12D3KooWBCzniwFvfS67HyZsJtiehd7RHtcKanGGCd8h6bjVWnkb`|`/ip4/5.199.168.17`|
-|`12D3KooWNbvFFq8kzS3eMonQYEmFyWTHboMZnHKr4XnN53gJKqhH`|`/ip4/5.199.162.220`|
+|`bafzbeibhqavlasjc7dvbiopygwncnrtvjd2xmryk5laib7zyjor6kf3avm`|`/dns4/elastic.dag.house/tcp/443/wss`|
```

Comment:
@alanshaw Can you confirm that this is correct?

Reply:
👍 thats correct